addon/components/object-list-view.js

```diff
@@ -502,6 +502,10 @@ export default FlexberryBaseComponent.extend(
     }
 
     let cols = this._generateColumns(projection.attributes);
+    if ('caption' in projection) {  //GroupEdit has caption values, dont use userSettings, return basic order
```

Comment:
We write each comment on separate line.
There should be a dot at the end of the comment and a space after "//".